README.md

```diff
@@ -32,6 +32,8 @@ This dbt package contains models, macros, seeds, and tests for [Fullstory](https
 | fullstory_event_types | A list of event types to auto-generate rollups for in the `users` and `sessions` model. |
 
 > We **highly recommend** using `fullstory_events_database`, `fullstory_events_schema` and `fullstory_events_table` to indicate the location of the Fullstory events table that is synced from Data Destinations. Using these variables allow you to use a separate database or schema for the Fullstory events table than your dbt package.
+>
+> We **highly recommend** utilizing the Fullstory OrgID as the `target` name in `profiles.yml`. This will not only make Umbrella configurations easier but will also allow for `replay_url.sql` to function as intended.
```

Comment:
Let's remove this from this PR for the moment. The placement of this guidance is a bit off given it's in the `Vars` section. I don't want to block the new field update on getting this right, however.

Reply:
That should be undone now, thanks for pointing it out!